stl/inc/algorithm

```diff
@@ -4408,7 +4408,7 @@ template <class _ExPo, class _BidIt, class _FwdIt, _Enable_if_execution_policy_t
 _FwdIt reverse_copy(_ExPo&&, _BidIt _First, _BidIt _Last, _FwdIt _Dest) noexcept /* terminates */ {
     // copy reversing elements in [_First, _Last)
     // not parallelized as benchmarks show it isn't worth it
-    _REQUIRE_PARALLEL_ITERATOR(_FwdIt);
+    _REQUIRE_MUTABLE_PARALLEL_ITERATOR(_FwdIt);

```

Comment:
We previously didn't check `_BidIt`, presumably because input iterators couldn't be silently passed to something expecting bidi operations.

Now, should we explicitly check `_REQUIRE_CONST_PARALLEL_ITERATOR(_BidIt)`?

Reply:
Almost certainly.